### DISCUSSION TITLE
Dialog open speed very slow

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#6666] [Q&A] [answered]
I am building an app that requires reading file path. 

The tauri [dialog open api](https://tauri.app/v1/api/js/dialog/#open) works but took 2 seconds to open the window, which is unusal comparing to a normal browser.

It doesn't seem to a problem with the frontend framework. I tried with both nuxt and sveltekit project, and both opens slowly.

**Comment by FabianLars** [comment]
Can you open an Issue instead of a Discussion for this please? You're not the first one mentioning this iirc

**Reply by HuakunShen** [reply]
OK.

**Comment by HuakunShen** [comment, marked as answer]
Converted to an issue https://github.com/tauri-apps/tauri/issues/6675
